soaks/bin/analyze_experiment

```diff
@@ -117,7 +117,14 @@ print("The goal of these tests are to determine, quickly, if vector performance
 print("Test units below are bytes/second/CPU, except for 'skewness'. The", end=' ')
 print("further 'skewness' is from 0.0 the more indication that vector lacks", end=' ')
 print("consistency in behavior, making predictions of fitness in the field challenging.")
+print()
+print("The table below lists those experiments that have experienced a", end=' ')
+print("statistically significant change in their throughput performance between", end=' ')
+print("baseline and comparision SHAs, with {}% confidence. Negative values mean".format((1.0 - args.p_value) * 100), end=' ')
+print("that baseline is faster, positive comparison. Results that do not exhibit more than a ±{}%".format(args.mean_drift_percentage), end=' ')
+print("change in mean throughput are discarded.")
 print("</details>")
+print()
```

Comment:
So this isn't quite correct, I don't think. The paragraph here won't make sense if there are no changes, considering that the table being referenced doesn't exist. A rewording to describe the sections that follow and their purpose would work though, for this non-conditional approach.